apm-collector/apm-collector-worker/src/main/java/org/skywalking/apm/collector/worker/globaltrace/GlobalTraceGetWithGlobalId.java

```diff
@@ -33,14 +32,13 @@ public void preStart() throws ProviderNotFoundException {
         getClusterContext().findProvider(GlobalTraceSearchWithGlobalId.WorkerRole.INSTANCE).create(this);
     }
 
-    @Override
-    protected void onSearch(Map<String, String[]> request, JsonObject response) throws Exception {
-        if (!request.containsKey("globalId")) {
+    @Override protected void onReceive(Map<String, String[]> parameter, JsonObject response) throws Exception {
+        if (!parameter.containsKey("globalId")) {
             throw new IllegalArgumentException("the request parameter must contains globalId");
         }
-        logger.debug("globalId: %s", Arrays.toString(request.get("globalId")));
+        logger.debug("globalId: %s", Arrays.toString(parameter.get("globalId")));
```

Comment:
`Arrays.toString(parameter.get("globalId")` has too much cost. You should add an `isDebug` enable before `logger.debug` for considering perfomance.